views/annotations/mutation.jade

```diff
@@ -0,0 +1,192 @@
+									th Commenter
+									th Text
+									th Direction
+							tbody
+								-for (var i = 0; i < annotation.comments.length; i++)
+									-var comment = annotation.comments[i];
+										tr(class="comment-row")
+											td(class="comment-name") #{comment.name}
+											td(class="comment-body") #{comment.comment}
+											td(class="comment-dir") #{comment.direction}
+				hr(style="clear:both")
+				div(id="commentVote", class="container", style="width%100;")
+					h4 Vote (submitting without a vote deletes comment):
+					textarea(class="form-control", id="mutation-comment", placeholder="Enter any comments.", type="textarea")
+					| Vote: 
+					a(href="#", class="upvote" style="color:#000") &#x25B2;&nbsp;
+					a(href="#", class="downvote" style="color:#000") &#x25BC;
+					button(type="submit", id="submit", class="btn btn-default") Submit
+				div(id="annotationStatus", class="text-center", style="width:100%;display:off")
+
+block belowTheFold
+	script(src='/components/DataTables/media/js/jquery.dataTables.min.js').
+	script(src='/js/dataTableWrapper.js').
+	script(type='text/javascript').
+		//- Convert the tables into DataTables
+		addDataTable({ tableID: "#variant-table", aaSorting: [[0, "asc"]] });
+		function annotationStatus(msg, classes) {
+			$('#annotationStatus').attr('class', classes);
+			$('#annotationStatus').html(msg);
+			$('#annotationStatus').show().delay(5000).slideUp();
+		}
+		var infoClasses	 = 'alert alert-info',
+			warningClasses = 'alert alert-warning',
+			successClasses = 'alert alert-success';
+
+		$(document).ready(function() {
+		
+			var my_comment = $('.comment-name').filter(function () { 
+				return $.text([this]) != 'Anonymous';
+			});
+
+			if (my_comment.length > 0) {
+				$('textarea#mutation-comment').val(my_comment.siblings('.comment-body').text());
+				var voteDir = my_comment.siblings('.comment-dir').text();
+				if (voteDir === 'up') {
+					$('a.upvote').addClass('upvote-on');
+				} else if (voteDir === 'down') {
+					$('a.upvote').addClass('upvote-on');
+				}
+			}
+			
+			$('a.upvote').on('click', function(){
+				var sibling = $(this).siblings('a');
+				$(this).toggleClass('upvote-on');
+				if(sibling.hasClass('downvote-on')) {
+					sibling.removeClass('downvote-on');
+				}
+			});;			
+			$('a.downvote').on('click', function(){
+				var sibling = $(this).siblings('a');
+				$(this).toggleClass('downvote-on');
+				if(sibling.hasClass('upvote-on')) {
+					sibling.removeClass('upvote-on');
+				}
+			});
+
+			$('button#submit').on('click', function() {
+				var vote = 'remove';
+				if ($('a.upvote').hasClass('upvote-on')) {
+					vote = 'up';
+				} else if ($('a.downvote').hasClass('downvote-on')) {
+					vote = 'down';
+				}
+
+				var formData = new FormData();
+				var uid = $('#annotation-table').attr('annotation-id');
+				var comment_body = $('textarea#mutation-comment').val() ? $('textarea#mutation-comment').val() : "<empty>";
```

Comment:
I'm not sure it makes sense to actually store "<empty>" in the database. One reason not to is that when you reload the page, your comment is already preloaded as "<empty>" (which is kind of strange):
![screen shot 2015-09-13 at 11 03 00 am](https://cloud.githubusercontent.com/assets/2043206/9837255/1904bbb6-5a07-11e5-8b89-d6ec59331cec.png)